app/services/piv_cac_service.rb

```diff
@@ -61,26 +60,23 @@ def token_decoded(token)
     end
 
     def token_response(token)
-      # FUTURE: For consistency replace raw NET::HTTP with Faraday equivalent.
-      http = Net::HTTP.new(verify_token_uri.hostname, verify_token_uri.port)
-      http.use_ssl = verify_token_uri.scheme == 'https'
-      http.verify_mode = OpenSSL::SSL::VERIFY_NONE if FeatureManagement.identity_pki_local_dev?
-      http.start do |post|
-        post.request(decode_request(verify_token_uri, token))
+      # Assume ssl is off unless verify_token_uri uses https
+      ssl_config = false
+      if (verify_token_uri.scheme == 'https')
```

Comment:
this is ruby! We don't need these parens

Reply:
Yeah, force of habit and the linter didn't remind me the first time because of a missing space...thanks linter.

Reply:
to be fair, we don't need the vast majority of parens that we seem to want to have everywhere...

Reply:
I don't feel strongly, but I'll go on the record as liking parentheses 👀